application/shared-kernel/InfrastructureCore/InfrastructureCoreConfiguration.cs

```diff
@@ -25,11 +26,13 @@ public static class InfrastructureCoreConfiguration
         string connectionName
     ) where T : DbContext
     {
-        var connectionString = IsRunningInAzure
-            ? Environment.GetEnvironmentVariable("DATABASE_CONNECTION_STRING")
```

Comment:
Where is the `DATABASE_CONNECTION_STRING` moved to?

On my phone so maybe it just my screen that is small :)

Reply:
Hm I assume that that we don't need it anymore because the connection should be done just by using the database name that we set up with Aspire. But of course I will test it.

Reply:
In Azure it will get the connection string from the `DATABASE_CONNECTION_STRING`.